app/helpers/application_helper/button/template_provision.rb

```diff
@@ -1,10 +1,6 @@
 class ApplicationHelper::Button::TemplateProvision < ApplicationHelper::Button::Basic
-  def calculate_properties
-    super
-    self[:title] = _("Selected item is not eligible for Provisioning") if disabled?
-  end
-
   def disabled?
-    !@record.supports_provisioning?
+    @error_message = _('Selected item is not eligible for Provisioning') unless @record.supports_provisioning?
+    @error_message.present?
```

Comment:
[GenericFeatureButtonWithDisable](https://github.com/ManageIQ/manageiq/blob/master/app/helpers/application_helper/button/generic_feature_button_with_disable.rb) can be used here